f/connectors/kobotoolbox/kobotoolbox_responses.py

```diff
@@ -469,12 +468,15 @@ def handle_output(self, all_submissions):
         all_submissions : list of dict
         """
         table_name = self.table_name
+        columns_table_name = (
```

Comment:
Add a code comment highlighting that we are NOT using the "uniqify" feature of this function (by passing the empty set of existing names).

Or else don't call that function at all (I'd argue it's a bit misleading to do so) and instead just

```suggestion
        columns_table_name = f"{table_name[:54]}__columns"
```

Note that if you had two very long responses tables with similar prefixes, they might clobber each other's column tables.  It's not the end of the world, as I assume they should nonetheless end up remapped correctly!

Reply:
Ah, much more straightforward, like `slice` in javascript. Great, thanks.